backend/server/src/test/resources/migrations/testing/h2/1_release_2022_08.up.sql

```diff
@@ -0,0 +1,1154 @@
+
+-- H2 2.1.214;
+SET DB_CLOSE_DELAY -1;
+;
+CREATE USER IF NOT EXISTS "SA" SALT '60a1b60063febe69' HASH '6be81f507cd19d4282527ccfd086ef287403aea65656c2adcae67979698268bd' ADMIN;
+CREATE SEQUENCE "HIBERNATE_SEQUENCE" START WITH 1;
+CREATE MEMORY TABLE "DATABASE_CHANGE_LOG_LOCK"(
+    "ID" INTEGER NOT NULL,
+    "LOCKED" BOOLEAN NOT NULL,
+    "LOCKGRANTED" TIMESTAMP,
+    "LOCKEDBY" CHARACTER VARYING(255)
+);
+ALTER TABLE "DATABASE_CHANGE_LOG_LOCK" ADD CONSTRAINT "PK_DATABASE_CHANGE_LOG_LOCK" PRIMARY KEY("ID");
+-- 1 +/- SELECT COUNT(*) FROM DATABASE_CHANGE_LOG_LOCK;
+INSERT INTO "DATABASE_CHANGE_LOG_LOCK" VALUES(1, FALSE, NULL, NULL);
+CREATE MEMORY TABLE "DATABASE_CHANGE_LOG"(
+    "ID" CHARACTER VARYING(255) NOT NULL,
+    "AUTHOR" CHARACTER VARYING(255) NOT NULL,
+    "FILENAME" CHARACTER VARYING(255) NOT NULL,
+    "DATEEXECUTED" TIMESTAMP NOT NULL,
+    "ORDEREXECUTED" INTEGER NOT NULL,
+    "EXECTYPE" CHARACTER VARYING(10) NOT NULL,
+    "MD5SUM" CHARACTER VARYING(35),
+    "DESCRIPTION" CHARACTER VARYING(255),
+    "COMMENTS" CHARACTER VARYING(255),
+    "TAG" CHARACTER VARYING(255),
+    "LIQUIBASE" CHARACTER VARYING(20),
+    "CONTEXTS" CHARACTER VARYING(255),
+    "LABELS" CHARACTER VARYING(255),
+    "DEPLOYMENT_ID" CHARACTER VARYING(10)
+);
+-- 336 +/- SELECT COUNT(*) FROM DATABASE_CHANGE_LOG;
+INSERT INTO "DATABASE_CHANGE_LOG" VALUES('createTable-1', 'anandJ', 'liquibase/create-tables-changelog-1.0.xml', TIMESTAMP '2022-11-10 15:43:59.783151', 1, 'EXECUTED', '8:b64eccfe76381356d89452ebadacfdf7', 'createTable tableName=artifact_store; createTable tableName=comment; createTable tableName=dataset; createTable tableName=git_snapshot; createTable tableName=job; createTable tableName=path_dataset_version_info; createTable tableName=query_dataset...', '', NULL, '4.9.1', NULL, NULL, '8075239297');
+INSERT INTO "DATABASE_CHANGE_LOG" VALUES('create_artifact_store', 'anandJ', 'liquibase/create-tables-changelog-1.0.xml', TIMESTAMP '2022-11-10 15:43:59.801159', 2, 'MARK_RAN', '8:a1d122741f130094a332b6b93e3687e8', 'createTable tableName=artifact_store', '', NULL, '4.9.1', NULL, NULL, '8075239297');
+INSERT INTO "DATABASE_CHANGE_LOG" VALUES('create_comment', 'anandJ', 'liquibase/create-tables-changelog-1.0.xml', TIMESTAMP '2022-11-10 15:43:59.803158', 3, 'MARK_RAN', '8:b41aeb46a59fe79e4bc90d2ac24ba787', 'createTable tableName=comment', '', NULL, '4.9.1', NULL, NULL, '8075239297');
```

Comment:
whoa. I don't know that we need to be inserting anything into liquibase any more!

Reply:
especially for H2, where this data will definitely never be used.

Reply:
Ok. I will remove it.